ffhttpd.c

```diff
@@ -28,6 +28,7 @@ typedef int (*PFN_CGI_MAIN)(char *request_type, char *request_path, char *url_ar
 #include <netinet/in.h>
 #define  closesocket close
 #define  SOCKET      int
+#define _snprintf snprintf

```

Comment:
I think it's better to change _snprintf to snprintf in function handle_http_request.

Reply:
I used add this macro because below gcc warnning
> warning: implicit declaration of function ‘_snprintf’; did you mean ‘vsnprintf’

after reading [ms docs about _snprintf](https://docs.microsoft.com/en-us/cpp/c-runtime-library/reference/snprintf-snprintf-snprintf-l-snwprintf-snwprintf-l?view=msvc-160)  I will find another way to surrpress the gcc warnnings

> The snprintf function and the _snprintf family of functions format and store count or fewer characters in buffer. The snprintf function always stores a terminating null character, truncating the output if necessary. The _snprintf family of functions only appends a terminating null character if the formatted string length is strictly less than count characters. Each argument (if any) is converted and is output according to the corresponding format specification in format. The format consists of ordinary characters and has the same form and function as the format argument for printf. If copying occurs between strings that overlap, the behavior is undefined.